bin/pygrb/pycbc_pygrb_efficiency

```diff
@@ -239,18 +241,12 @@ if onsource_file:
 
     # Calculate BestNRs and record loudest trig by BestNR
     if on_trigs:
-        on_trigs_bestnrs = ppu.get_bestnrs(on_trigs,
-                                           q=chisq_index, n=chisq_nhigh,
-                                           null_thresh=null_thresh,
-                                           snr_threshold=snr_thresh,
-                                           sngl_snr_threshold=sngl_snr_thresh,
-                                           chisq_threshold=new_snr_thresh,
-                                           null_grad_thresh=null_grad_thresh,
-                                           null_grad_val=null_grad_val)
+        on_trigs_bestnrs = on_trigs['network/reweighted_snr'][:]
         loud_on_bestnr_trigs, loud_on_bestnr = \
             np.asarray([[x, y] for y, x in sorted(zip(on_trigs_bestnrs, on_trigs),
                                                   key=lambda elem: elem[0],
                                                   reverse=True)])[0]
```

Comment:
I don't understand how this would work. You are zipping an array of reweighed SNRs with `on_trigs` which is an object returned by opening and hdf5 file, so it looks like `<KeysViewHDF5 ['H1', 'L1', 'V1', 'network']>`. What you want is to sort the content of each key (['H1', 'L1', 'V1', 'network']) on the basis of the reweighed SNRs.

Reply:
Ah I see, I'll have to look at this one a bit more.